### PR TITLE
`Paywalls`: add support for displaying `StoreProductDiscount`s

### PR DESCRIPTION
**Pull request by NachoSoto**
image

### CODE REVIEW

**Comment by NachoSoto** [comment]
Oh gotta add a check for whether the user is eligible.

**Comment by NachoSoto** [comment]
> Oh gotta add a check for whether the user is eligible.

Doing that in a separate PR.

**Review by joshdholtz — APPROVED** [review]
💯 LGTM

File: RevenueCatUI/Helpers/ProcessedLocalizedConfiguration.swift [hunk 1]
```diff
@@ -6,7 +6,9 @@ struct ProcessedLocalizedConfiguration: PaywallLocalizedConfiguration {
     var title: String
     var subtitle: String
     var callToAction: String
+    var callToActionWithIntroOffer: String
     var offerDetails: String
+    var offerDetailsWithIntroOffer: String
```
Comment (aboedo) [inline RevenueCatUI/Helpers/ProcessedLocalizedConfiguration.swift:11]: maybe these should be optional, not all products have an intro offer

Comment (NachoSoto) [reply RevenueCatUI/Helpers/ProcessedLocalizedConfiguration.swift:11]: Doing in the next PR 👍🏻

**Pull request merged**